.gitignore

```diff
@@ -1,5 +1,4 @@
 /site/js/function-plot.js
-/dist
```

Comment:
The contents of `dist/` are autogenerated, maybe during the build you could copy .d.ts files from `src/` to this folder through webpack instead?